templates/webapi/layouts/rate_limit_error.html.ep

```diff
@@ -0,0 +1,8 @@
+% layout 'bootstrap';
```

Comment:
Why is this template in the `layouts` directory? Shouldn't it be in the directory above?